.idea/runConfigurations/Simple_topology.xml

```diff
@@ -16,6 +16,7 @@
     </extension>
     <method v="2">
       <option name="Make" enabled="true" />
+      <option name="BSP.BeforeRunTask" enabled="true" />
```

Comment:
unclear why we have this file check-in at all ... let me ask on Slack

Reply:
Seems unused: https://daholdings.slack.com/archives/C03J6NCPSE4/p1757940888445849

Let's wait and drop in a separate PR together with the .gitignore exclude once people had time to shout.